src/app-deploy.spec.ts

```diff
@@ -36,7 +36,7 @@ describe('deploy-app', () => {
     expect(app.updatable).toBe(creationMetadata.updatable)
     expect(app.deletable).toBe(creationMetadata.deletable)
     expect(app.version).toBe(creationMetadata.version)
-  })
+  }, 20_000)
```

Comment:
These overcome the intermittent indexer slowdown that exists on current localnet